src/styles/index.scss

```diff
@@ -1,3 +1,218 @@
+@import './variables';
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  text-transform: uppercase;
+  padding: 0 50px;
+  height: 60px;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+
+  &__logo-image {
+    height: 40px;
+    width: 40px;
+    margin-top: 2px;
+  }
+}
+
+.nav {
+  &__list {
+    list-style-type: none;
+    display: flex;
+    margin: 0;
+    padding: 0;
+  }
+
+  &__item {
+    margin-left: 20px;
+
+    &:first-child {
+      margin-left: 0;
+    }
+
+    &:last-child {
+      margin-right: 0;
+    }
+  }
+
+  &__link {
+    text-decoration: none;
+    font-family: Roboto, sans-serif;
+    font-size: 12px;
+    font-weight: 500;
+    white-space: nowrap;
+    color: $main-color;
+    text-align: center;
+    align-items: center;
+    height: 60px;
+    display: flex;
+    position: relative;
+
+    &:hover {
+      color: $third-color;
+    }
+
+    &--active {
+      color: $third-color;
+      position: relative;
+
+      &::after {
+        content: '';
+        display: block;
+        background-color: $third-color;
+        position: absolute;
+        width: 100%;
+        height: 4px;
+        border-radius: 8px;
+        bottom: 0;
+        left: 0;
+      }
+    }
+  }
+}
+
+.container {
+  padding: 50px 40px;
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(200px, 200px));
+  justify-content: center;
+  gap: 46px 48px;
+}
+
+.card {
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: $card-width;
+  padding: 32px 16px 16px;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
+
+  &__image {
+    background-image: url(../images/imac.jpeg);
+    background-size: 160px 134px;
+    height: 134px;
+    width: 160px;
+    margin-bottom: 40px;
+  }
+
+  &__name {
+    width: 100%;
+    color: $main-color;
+    font-size: 12px;
+    font-weight: 500;
+    font-family: inherit;
+    line-height: 18px;
+    margin-bottom: 4px;
+  }
+
+  &__code {
+    font-size: 10px;
+    font-weight: 400;
+    font-family: inherit;
+    line-height: 14px;
+    width: 100%;
+    margin-bottom: 16px;
+    color: $second-color;
+  }
+
+  &__review {
+    height: 16px;
+    margin-bottom: 24px;
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__price {
+    display: flex;
+    width: 100%;
+    margin-bottom: 16px;
+    justify-content: space-between;
+    align-items: center;
+    height: 18px;
+
+    &-text {
+      font-size: 12px;
+      font-family: inherit;
+      color: $second-color;
+    }
+
+    &-sum {
+      font-size: 16px;
+      font-weight: 700;
+      font-family: inherit;
+      color: $main-color;
+      line-height: 18px;
+    }
+
+    &:hover &__name {
+      color: #34568b;
+    }
```

Comment:
The `&:hover &__name` selector seems misplaced. It targets `.card__name` when `.card__price` is hovered, which might not be the intended behavior. Ensure this is correct or adjust the selector to target the intended element.